backend/src/model/entity/map_impl.rs

```diff
@@ -15,8 +19,12 @@ use crate::{
 use super::{Map, NewMap};
 
 impl Map {
-    /// Get a page of maps.
-    /// Can be filtered by its active status if one is provided in `search_parameters`.
+    /// Get the top plants matching the search query.
+    ///
+    /// Can be filtered by active status and owner id if provided in `search_parameters`.
```

Comment:
In which ways? Please describe differences to plant search (if there are, otherwise say it works identical).